site2/docs/concepts-messaging.md

```diff
@@ -396,6 +397,57 @@ producer = client.create_producer(topic='my-topic', batching_type=pulsar.Batchin
 > * You need to specify a key or orderingKey for messages.
 > * You cannot use cumulative acknowledgment with Key_Shared type.
 
+### Subscription mode
+
+#### What is subscription mode
+
+The subscription mode indicates the cursor type. 
+
+- When a subscription is created, an associated cursor is created to record the last consumed position. 
+
```

Comment:
I think no blank line is needed for a list in markdown.

```
- line 01
- line 02
```

Reply:
Yes, but I insert a blank line to make things more clear.